sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/WebPubSubServiceClient.cs

```diff
@@ -601,6 +609,7 @@ internal virtual Response GroupExistsImpl(string group, RequestContext context =
         /// <param name="reason"> The reason closing the client connection. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="group"/> is null. </exception>
+        /// <exception cref="ArgumentException"> <paramref name="group"/> is an empty string, and was expected to be non-empty. </exception>
```

Comment:
Can we make the doc comment for `ArgumentException` similar to the one for `ArgumentNullException`, meaning remove the part ", and was expected to be non-empty".

Reply:
We chose this because best practice for error messages is that they a) tell the user what went wrong and b) tell the user how to fix it.

Reply:
Not disagreeing in general about it. I wonder if users are getting any additional information with the message in this particular case. The first part already states the issue "<param> is an empty string" and the fact that an argument exception was thrown means that is an error.

I see it similar to the `ArgumentNullException` message. We don't have to add "and was expected to be non-null" to the message.